.gitignore

```diff
@@ -1,3 +1,6 @@
 /oci-create-runtime-bundle
 /oci-unpack
 /oci-image-validate
+
+*.1
+*.patch
```

Comment:
I use `*.patch` files in projects where contribution is by mailing patches to a mailing list.  But I personally don't use them much in push-based projects like those on GitHub.  If you generate `*.patch` files as part of your personal workflow with this project, I think you should add a local ignore rule (gitignore(5) gives `$HOME/.config/git/ignore` and `$GIT_DIR/info/exclude` as possible locations).  But if rules in the committed `.gitignore` are based on workflow idiosyncrasies, it becomes hard to maintain them.